doc/container.md

```diff
@@ -240,15 +248,12 @@ Upgrading a Container Image
 The applications in your container are an active part of the system as a
 whole, so make it a routine to keep your container images up-to-date!
 
-> **Note:** the default writable layer is lost when upgrading the image.
-> Use named volumes for content that you want to persist across upgrades.
+Containers are locked to the image hash at the time of creation.  This
+applies to all images, not just those tagged `:edge` or `:latest`.  To
+upgrade a versioned image is obvious: update the `running-config` to use
```

Comment:
It was not obvious to yours truly, and now I feel stupid :smile:, maybe drop that word?

Reply:
Good point, we should always avoid words like that, nothing is "obvious" or "simple", or "simply do" ...